config/rbac/role.yaml

```diff
@@ -218,6 +218,14 @@ rules:
   - list
   - update
   - watch
+- apiGroups:
```

Comment:
Duplicate of L229

Reply:
not really? L229+ is anyuid+privileged, still used by OVNController. This file is auto-updated by operator make hooks.

Reply:
I updated the title here + clarified in commit messages what's actually happening here + split the single commit into three, separating concerns. Let me know if this helps.

Reply:
Thanks Ihar, yes right these are generated from annotations defined in *_controllers.go

Still it's doubtful if the definitions from this file is really used since the introduction of Dynamic roles/serviceaccounts, can check with @dprince on this.

Reply:
> Still it's doubtful if the definitions from this file is really used since the introduction of Dynamic roles/serviceaccounts, can check with @dprince on this.

Ok checked this in a test patch https://github.com/openstack-k8s-operators/ovn-operator/pull/80 and those are needed to create dynamic role/service_accounts. A service account/user can create role only if it had the same or higher permissions https://kubernetes.io/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update